src/Marketplace/MarketplaceScreen.js

```diff
@@ -22,9 +30,21 @@ const styles = StyleSheet.create({
     marginTop: 10,
     backgroundColor: '#0492c2',
   },
+  card: {
+    width: '90%',
+  },
 });
 
 export default function MarketplaceScreen() {
+  const [card, setCard] = useState([]);
+
+  useEffect(() => {
+    base('APIFM Produce Cards').find('recuOS5skQ04lUxBk', (err, record) => {
```

Comment:
where is this base and this record? I'm getting that card is not set and card.fields is undefined.